packages/next/src/shared/lib/app-router-context.shared-runtime.ts

```diff
@@ -69,7 +69,6 @@ export type CacheNode =
 
 export interface NavigateOptions {
   /** @internal */
```

Comment:
I think this should have been removed too, I've opened a PR here: https://github.com/vercel/next.js/pull/59001